flask_wtf/file.py

```diff
@@ -91,3 +91,33 @@ def __call__(self, form, field):
 
 
 file_allowed = FileAllowed
+
+class FileMaxSize(object):
+    """Validates that the uploaded file is within a maximum file size given in bytes
+
+    :param max_size: maximum file size given in kilobytes
+    :param message: error message
+
+    You can also use the synonym ``file_max_size``.
+    """
+
+    def __init__(self, max_size, message=None):
+        self.max_size = max_size
+        self.message = message
+
+    def __call__(self, form, field):
+        if not (isinstance(field.data, FileStorage) and field.data):
```

Comment:
I presume this was added since it occurs in other validation classes ([example](https://github.com/lepture/flask-wtf/blob/30f90d70864864960bae8643947dfcd416f1f871/flask_wtf/file.py#L74-L75)) - @davidism is this not desirable here?